packages/vscode-extension/src/webview/components/DeviceSelect.tsx

```diff
@@ -120,7 +126,18 @@ function DeviceSelect() {
     if (selectedDevice?.id !== value) {
       const deviceInfo = (devices ?? []).find((d) => d.id === value);
       if (deviceInfo) {
-        project.startOrActivateSessionForDevice(deviceInfo);
+        if (deviceInfo.deviceType === DeviceType.Tablet && hasAccessToProFeatures(licensedStatus)) {
```

Comment:
Surely if you have access, you don't want the paywall?
```suggestion
        if (deviceInfo.deviceType === DeviceType.Tablet && !hasAccessToProFeatures(licensedStatus)) {
```